### PR TITLE
Fixup CodeQL3000 injection

### PR DESCRIPTION
**Pull request by wtgodbe**

### CODE REVIEW

File: .config/tsaoptions.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+  "areaPath": "DevDiv\\ASP.NET Core",
```
Comment (wtgodbe) [inline .config/tsaoptions.json:2]: @rafikiassumani-msft or @danmoseley, what area path do you suggest for legacy SignalR❔

Comment (BrennanConroy) [reply .config/tsaoptions.json:2]: This isn't legacy, it's current

Comment (wtgodbe) [reply .config/tsaoptions.json:2]: I copy/pasted that from the PR in the other repo 😆

File: azure-pipelines.yml [hunk 2]
```diff
@@ -16,8 +16,19 @@ variables:
     value: true
   - name: _BuildConfig
     value: 'Debug'
+  # CodeQL3000 tasks are auto-injected into official builds of main. Variables below are ignored otherwise.
```
Comment (dougbu) [inline azure-pipelines.yml:19]: No, the tasks will be auto-injected into _all_ builds of main. Make `Codeql.Enabled` undefined or `false` in public builds. We don't want or need duplicate bugs.

File: .config/tsaoptions.json (hunk 1 above)
Comment (dougbu) [reply .config/tsaoptions.json:2]: suspect we're fine here unless @danmoseley or @rafikiassumani-msft has a better idea

File: azure-pipelines.yml [hunk 3]
```diff
@@ -16,10 +16,19 @@ variables:
     value: true
   - name: _BuildConfig
     value: 'Debug'
-  - name: Codeql.Enabled
-    value: true
-  # used for post-build phases, internal builds only
   - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - name: Codeql.Enabled
```
Comment (dougbu) [inline azure-pipelines.yml:20]: Not sure how big a deal it is but will get `CodeQL3000` tasks auto-injected into all build jobs. That's at best redundant.

File: .config/tsaoptions.json (hunk 1 above)
Comment (mkArtakMSFT) [reply .config/tsaoptions.json:2]: DevDiv\ASP.NET OOB\SignalR

**Review by BrennanConroy — APPROVED** [review]

**Pull request merged**